bessctl/conf/testing/test_constraint_checker.bess

```diff
@@ -0,0 +1,67 @@
+
+def queue_test():
```

Comment:
We don't check for soft constraint failures in sanity check, which is why I copied it here.